src/library/vicon_estimator.cpp

```diff
@@ -544,4 +597,9 @@ double RotationalEstimator::quaternionRotationMagnitude(
     return negative_rotation_return;
   }
 }
+
+void RotationalEstimator::makeCovarianceSymmetric(
+    Eigen::Matrix<double, 6, 6>* covariance) {
+  *covariance = (*covariance + covariance->transpose()) / 2;
```

Comment:
Hmmmm. OK. This means putting argument checks into the whole project. But I guess they should be there anyway. I'll do it.